### PR TITLE
Fix / rich text box require

### PR DESCRIPTION
**Pull request by maxkrasnov**
#2288, #1998
remove 'require' attribute from rich_text_box.blade.php, because:

> Chrome wants to focus on a control that is required but still empty so that it can pop up the message 'Please fill out this field'. However, if the control is hidden at the point that Chrome wants to pop up the message, that is at the time of form submission, Chrome can't focus on the control because it is hidden, therefore the form won't submit.
> 
> So, to get around the problem, when a control is hidden by javascript, we also must remove the 'required' attribute from that control.

### CODE REVIEW

File: resources/views/formfields/rich_text_box.blade.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<textarea @if($row->required == 1) required @endif class="form-control richTextBox" data-name="{{ $row->display_name }}"  name="{{ $row->field }}" id="richtext{{ $row->field }}">
+<textarea @if($row->required == 1) @endif class="form-control richTextBox" data-name="{{ $row->display_name }}"  name="{{ $row->field }}" id="richtext{{ $row->field }}">
```
Comment (fletch3555) [inline resources/views/formfields/rich_text_box.blade.php]: If you're going to remove `required`, you need to remove the empty `@if() ... @endif`` around it.

**Review by fletch3555 — CHANGES_REQUESTED** [review]

**Comment by fletch3555** [comment]
I'm not too familiar with the i18n functionality, but I don't believe this is the correct solution.  @akazorg, do you have any thoughts/suggestions?

File: resources/views/formfields/rich_text_box.blade.php (hunk 1 above)
Comment (maxkrasnov) [reply resources/views/formfields/rich_text_box.blade.php]: ouhc, in a hurry missed

**Comment by akazorg** [comment]
@fletch3555 I wasn't aware of that Chrome bug. 
If this solution works, then looks great :)
Thanks for the PR @maxkrasnov!

**Comment by fletch3555** [comment]
@akazorg, how do the i18n fields work? Does it keep the normal EN field on the page but hidden?  If so, this would fix it.  If not, this just removes the ability to make the field required.  That's why I'm not sure this was the best fix.

**Comment by maxkrasnov** [comment]
@fletch3555 , @akazorg maybe add custom check require on js for this? agree?

**Comment by akazorg** [comment]
@maxkrasnov sorry but I don't have an answer, need to test this.

@fletch3555 the way I18N works is using an hidden input for each translatable field. This field holds a JSON object containing all available translations for each input.
When the user select another language, the JS will update all hidden fields with the previous language on screen.
Once User submits the form, the JS will again update hidden fields, and then submits the form.

**Comment by yourPixel** [comment]
@maxkrasnov Thank you very much !!!

**Pull request closed without merging**